pkg/cloudprovider/aws/builder.go

```diff
@@ -40,3 +41,19 @@ func NewCloudProvider(logger logr.Logger) (cloudprovider.CloudProvider, error) {
 
 	return p, nil
 }
+
+// NewGenericCloudProvider returns a new mock AWS cloud provider
+func NewGenericCloudProvider(autoscalingiface *fakeaws.Autoscaling, ec2iface *fakeaws.Ec2) cloudprovider.CloudProvider {
+	// autoscalingiface := &fakeaws.Autoscaling{
```

Comment:
Do we need to keep this if it's commented out?

Reply:
Good catch that was a mistake